build.gradle

```diff
@@ -132,9 +169,22 @@ publishing {
                 awsIm(AwsImAuthentication) // load from EC2 role or env var
             }
         }
+        maven {
+            name = "sonatype-staging"
+            url "https://oss.sonatype.org/service/local/staging/deploy/maven2"
+            credentials {
+              username ossrhUsername
+              password ossrhPassword
```

Comment:
replace with below to prevent gradle errors for folks who do not have this property configured and are not going to publish the artifacts. 
```
username project.hasProperty('ossrhUsername') ? project.property('ossrhUsername') : ''
password project.hasProperty('ossrhPassword') ? project.property('ossrhPassword') : ''
```

Reply:
Thanks for the suggestion. I will update it.